# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /home/peaceman/workspace/zf/tests/Zend/Text
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: TableTest.php
--- TableTest.php Base (BASE)
+++ TableTest.php Locally Modified (Based On LOCAL)
@@ -267,6 +267,17 @@
         $this->assertEquals($row->render(array(10, 10), $decorator), "│foo       │foobar    │\n│bar       │          │\n");
     }
 
+    public function testUnicodeRowMultiLine()
+    {
+        $decorator = new Zend_Text_Table_Decorator_Unicode();
+
+        $row = new Zend_Text_Table_Row();
+        $row->appendColumn(new Zend_Text_Table_Column("föö\nbär"));
+        $row->appendColumn(new Zend_Text_Table_Column("fööbär"));
+
+        $this->assertEquals($row->render(array(3, 10), $decorator), "│föö│fööbär    │\n│bär│          │\n");
+    }
+
     public function testTableConstructInvalidColumnWidths()
     {
         try {
